### PR TITLE
Output cucumber commands executed for easier debug

### PR DESCRIPTION
**Pull request by liangxia**
Related to https://github.com/openshift/release/pull/32646#issuecomment-1261739602

/cc @zhaozhanqi @weliang1 @jhou1

### CODE REVIEW

File: ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh [hunk 1]
```diff
@@ -34,6 +34,7 @@ else
 fi
 
 cd verification-tests
+echo "tags: ${UPGRADE_PRE_RUN_TAGS} and ${UPGRADE_SKIP_TAGS}"
```
Comment (zhaozhanqi) [inline ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh:37]: s/${UPGRADE_PRE_RUN_TAGS}/${UPGRADE_CHECK_RUN_TAGS}/

File: ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh [hunk 2]
```diff
@@ -34,6 +34,7 @@ else
 fi
 
 cd verification-tests
+echo "tags: ${UPGRADE_CHECK_RUN_TAGS} and ${UPGRADE_SKIP_TAGS}"
```
Comment (jhou1) [inline ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh:37]: Can we print the raw cucumber command here if we want them for debugging? e.g.

```
set -x
cucumber xxx
set +x
```

File: ci-operator/step-registry/cucushift/upgrade/prepare/cucushift-upgrade-prepare-commands.sh [hunk 3]
```diff
@@ -29,6 +29,7 @@ export OPENSHIFT_ENV_OCP4_USER_MANAGER_USERS=${USERS}
 export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/upgrade-prepare"
 
 cd verification-tests
+echo "tags: ${UPGRADE_PRE_RUN_TAGS} and ${UPGRADE_SKIP_TAGS}"
```
Comment (jhou1) [inline ci-operator/step-registry/cucushift/upgrade/prepare/cucushift-upgrade-prepare-commands.sh:32]: Same here

File: ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh (hunk 2 above)
Comment (jhou1) [reply ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh:37]: Or print the cucumber string command

**Pull request merged**